### ISSUE TITLE
Csv reports have filename too long for Excel

### ISSUE THREAD

**Issue by efronbs** [yahoo/fili#825]
Fili does not currently enforce a max file path length for downloaded csv reports. This is causing issues for some customers that try to open the reports in Microsoft Excel.

The solution is to enforce a max file path length, preferably one that works with Excel.

**Comment by kevinhinterlong** [comment]
This already got merged, but the [current implementation](
https://github.com/yahoo/fili/blob/628c8247fcca388ee14157316b23f385c5e15340/fili-core/src/main/java/com/yahoo/bard/webservice/web/util/ResponseUtils.java#L46-L48) doesn't preserve the `.csv` extension. Is this the intended behavior? If not maybe something like

```java
String rawFilePath = uriPath + interval;
int maxLength = MAX_EXCEL_FILE_PATH_LENGTH - CSV_EXT.length();
String truncatedRawFilePath = rawFilePath.substring(0, Math.min(rawFilePath.length(), maxLength));
String filename = truncatedRawFilePath + CSV_EXT;
return "attachment; filename=" + filename;
```

**Comment by kevinhinterlong** [comment]
Fixed by #828
